src/extension/openFileAtLocation.ts

```diff
@@ -70,7 +71,8 @@ function getPathForRNParentWorkspace(dir: string): Q.Promise<string> {
             return dir;
         }
         if (dir === "" || dir === "." || dir === "/" || dir === path.dirname(dir)) {
-            return Q.reject<string>(ErrorHelper.getInternalError(InternalErrorCode.WorkspaceNotFound, "React Native project workspace not found."));
+            const extraArgs = [{argument: "React Native project workspace not found.", isPii: false}];
```

Comment:
Do we really need to pass error message as argument?

Reply:
I added error with this code to the `errorStrings.ts` so  would be processed now as part of the WorkspaceNotFound error string